lib/screens/app/app.dart

```diff
@@ -136,6 +137,20 @@ class _AppState extends State<App> with WidgetsBindingObserver {
       var action = request.actions.first;
       var data = Map<String, dynamic>.from(action.data);
 
+      if (action.account == "token.seeds" &&
```

Comment:
An inline comment explaining this logic
Or move the logic to a variable with a meaningful name

if(accountIsReady) {
Navigate()
}